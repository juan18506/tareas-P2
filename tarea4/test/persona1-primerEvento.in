# creo Persona con 3 eventos
crearAgenda
crearPersona 1 2 Maria

crearFecha 22/3/2045
crearEvento 1 Evento1
agregarEventoPersona

crearFecha 20/3/2045
crearEvento 2 Evento2
agregarEventoPersona

crearFecha 20/3/2020
crearEvento 3 Evento3
agregarEventoPersona

# los eventos quedan en orden 3, 2, 1
imprimirPersona

# imprimo el evento 3
imprimirPrimerEventoPersona

# remuevo el ultimo evento
removerEventoPersona 1

# imprimo el evento 3
imprimirPrimerEventoPersona

# remuevo el primer evento
removerEventoPersona 3

# imprimo el evento 2
imprimirPrimerEventoPersona

liberarPersona

Fin