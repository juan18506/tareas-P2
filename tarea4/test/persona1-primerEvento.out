1>#  creo Persona con 3 eventos.
2>3>4>5>6>7>8>9>10>11>12>13>#  los eventos quedan en orden 3, 2, 1.
14>Persona 1: Maria, 2 aÃ±os
Evento 3:  Evento3
Fecha: 20/3/2020
Evento 2:  Evento2
Fecha: 20/3/2045
Evento 1:  Evento1
Fecha: 22/3/2045
15>#  imprimo el evento 3.
16>Evento 3:  Evento3
Fecha: 20/3/2020
17>#  remuevo el ultimo evento.
18>19>#  imprimo el evento 3.
20>Evento 3:  Evento3
Fecha: 20/3/2020
21>#  remuevo el primer evento.
22>23>#  imprimo el evento 2.
24>Evento 2:  Evento2
Fecha: 20/3/2045
25>26>Fin.
